dockstore-webservice/src/main/java/io/dockstore/webservice/resources/MetadataResource.java

```diff
@@ -287,6 +287,10 @@ public Response getRunnerDependencies(
             return Response.noContent().build();
         }
         boolean unwrap = !("json").equals(output);
+        if (!PipHelper.validateSemVer(clientVersion)) {
+            throw new CustomWebApplicationException(String.format("Invalid value for client version: `%s`. Value must be like `1.13.0`)", clientVersion, PipHelper.SEM_VER_STRING),
```

Comment:
You no longer need the `PipHelper.SEM_VER_STRING` parameter. 

I'm not 100% sold on  `Value must be like 1.13.0`, but I can't think of a better alternative that's not overly verbose.

Reply:
Haven't, but it seems worth a look